include/vinecopulib/vinecop/implementation/class.ipp

```diff
@@ -19,29 +19,21 @@ namespace vinecopulib {
 //! independence.
 //! @param d the dimension (= number of variables) of the model.
 inline Vinecop::Vinecop(const size_t d)
-{
-  d_ = d;
-
-  // 0-truncated D-vine with variable order (1, ..., d)
-  vine_struct_ =
-    RVineStructure(tools_stl::seq_int(1, d), static_cast<size_t>(0));
-
-  // pair_copulas_ empty = everything independence
-  threshold_ = 0.0;
-  loglik_ = NAN;
-}
+  : Vinecop(RVineStructure(tools_stl::seq_int(1, d), static_cast<size_t>(0)))
+{}
 
 //! @brief creates a vine copula with structure specified by an RVineStructure
 //! object; all pair-copulas are set to independence.
 //! @param structure an RVineStructure object representing the structure of
 //! the vine.
 inline Vinecop::Vinecop(const RVineStructure& structure)
-{
-  d_ = structure.get_dim();
-  vine_struct_ = structure;
+  : d_(structure.get_dim())
+  , vine_struct_(structure)
```

Comment:
the , have to be on the next line ?

Reply:
That's what the clang-format does with Mozilla style.

Reply:
ok :)